test/soll-runtime-test/runtest.sh

```diff
@@ -0,0 +1,9 @@
+SOLL_BASE="../.."
+
+cp ${SOLL_BASE}/build/utils/ssvm-evmc/test/soll-runtime-test/libsoll_runtime_test.so ${PWD}
+cp ${SOLL_BASE}/build/utils/ssvm-evmc/tools/ssvm-evmc/libssvm-evmc.so ${PWD}
+
+ln -sf ${SOLL_BASE}/test/libyul .
+ln -sf ${SOLL_BASE}/test/yul .
+
+python soll-runtime-test.py
```

Comment:
Looks like you only copy and create new files without cleaning up after the `runtest.sh`?

Reply:
the soft link file remains static and the shared library would be removed